### ISSUE TITLE
question about using 'sep_example.tf'

### ISSUE THREAD

**Issue by wl3b10s** [andrewowens/multisensory#23]
when using 'sep_example.tf' to test the train procedure of shift_net.

tf file is from  "http://people.eecs.berkeley.edu/~owens/multisensory/sep_example.tf"

i got messege like:

 File "train_shift.py", line 3, in <module>
    shift_net.train(shift_params.shift_v1(num_gpus=3), [0,1,2], restore = False)
  File "multisensory-master/src/shift_net.py", line 315, in train
    model.make_model()
  File "/multisensory-master/src/shift_net.py", line 155, in make_model
    self.inputs = read_data(pr, self.gpus)
  File "/multisensory-master/src/shift_net.py", line 17, in read_data
    lambda : shift_dset.make_db_reader(
  File "/multisensory-master/src/tfutil.py", line 484, in on_cpu
    return f()
  File "/multisensory-master/src/shift_net.py", line 19, in <lambda>
    num_db_files = pr.num_dbs))
  File "/multisensory-master/src/shift_dset.py", line 283, in make_db_reader
    ims, flows, samples, sfs, labels, ytids = tf.train.batch(example_list[0], batch_size)
  File "/data/wen/anaconda2/lib/python2.7/site-packages/tensorflow/python/training/input.py", line 927, in batch
    name=name)
  File "/data/wen/anaconda2/lib/python2.7/site-packages/tensorflow/python/training/input.py", line 722, in _batch
    dequeued = queue.dequeue_many(batch_size, name=name)
  File "/data/wen/anaconda2/lib/python2.7/site-packages/tensorflow/python/ops/data_flow_ops.py", line 464, in dequeue_many
    self._queue_ref, n=n, component_types=self._dtypes, name=name)
  File "/data/wen/anaconda2/lib/python2.7/site-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 2418, in _queue_dequeue_many_v2
    component_types=component_types, timeout_ms=timeout_ms, name=name)
  File "/data/wen/anaconda2/lib/python2.7/site-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/data/wen/anaconda2/lib/python2.7/site-packages/tensorflow/python/framework/ops.py", line 2956, in create_op
    op_def=op_def)
  File "/data/wen/anaconda2/lib/python2.7/site-packages/tensorflow/python/framework/ops.py", line 1470, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

OutOfRangeError (see above for traceback): FIFOQueue '_1_batch/fifo_queue' is closed and has insufficient elements (requested 15, current size 0)

is there some setting in shift_params.py to change or set?

thanks.

**Comment by wl3b10s** [comment]
i guess this may be reason:
"
2019-03-22 15:45:37.811825: W tensorflow/core/framework/op_kernel.cc:1192] Invalid argument: Name: <unknown>, Feature: im_0 (data type: string) is required but could not be found.
2019-03-22 15:45:41.945426: W tensorflow/core/framework/op_kernel.cc:1192] Out of range: FIFOQueue '_1_batch/fifo_queue' is closed and has insufficient elements (requested 5, current size 0)
"

im_0 needed is not found.

**Comment by ruizewang** [comment]
Hello, have you slove the problem? I also meet the problem when try to train shift_model. I don't kown what the input is for training shift_model.
